### PR TITLE
Added history support.

### PR DESCRIPTION
**Pull request by gilescope**
Wasn't sure whether to parse created time so left that as an i64.
`<missing>` images got switched to None.
"" comments got switched to None.

### CODE REVIEW

**Comment by gilescope** [comment]
Will fmt + add test.

**Comment by eldesh** [comment]
Now, circleci settings have been fixed on the master branch. Please rebase.
// On commit:bded4ef, `build` job is failed with memory allocation error (by too parallelized compilation).

File: src/docker.rs [hunk 1]
```diff
@@ -650,6 +674,19 @@ impl Docker {
...
 
+    /// History of an image
+    ///
+    /// # API
+    /// /images/{name}/history
+    ///
+    pub fn history_image(&self, name: &str) -> Result<Vec<HistoryImage>> {
+        let result = self.http_client()
+            .get(self.headers(), &format!("/images/{}/history", name))
+            .and_then(api_result);
```
[the remark this answers is not in the document]
Comment (gilescope) [reply src/docker.rs]: Thanks, did something close to that but without scary &** things in. In my head &* cancels out to a no-op. I'm sure there's something I misunderstand there.

**Comment by gilescope** [comment]
All done?

**Comment by eldesh** [comment]
How about type of `HistoryImage::comment` ? Is `Option` still needed ?

**Comment by gilescope** [comment]
I wonder if there's a better name than 'HistoryImage'? How about 'ImageChange', then the history call returns Vec<ImageChange>? Or can we be clearer still - would it be correct to call each an 'ImageLayer'?

**Comment by gilescope** [comment]
Seems I need logs as well. Will do that as a separate PR.

**Comment by eldesh** [comment]
Looks good 👍

**Pull request merged**